refactron/core/inference.py

```
@@ -28,10 +30,67 @@ def parse_string(code: str, module_name: str = "") -> nodes.Module:
+        try:
+            with open(abs_path, "r", encoding="utf-8") as f:
+                code = f.read()
+
+            # Resolve module name to keep astroid's state consistent
+            modname = ""
+            try:
+                from astroid import modutils
+
+                modname = modutils.modname_from_path(abs_path)
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)